### ISSUE TITLE
Four suggestions regarding configs and balance

### ISSUE THREAD

**Issue by darklordwonton** [gigaherz/ElementsOfPower#17]
Two suggestions for config options:
1. The ability to disable ores from generating (in case you want to control it with another mod)
2. The ability to change the amount of essence each gem can hold (for balance)

Two balance suggestions:
1. Make charging faster, charging gems is pretty slow right now and I think this mod would be more fun to play with if you didn't have to wait a while to recharge your wand after casting a few spells
2. Make staves and wands roughly the same price but make staves able to hold 1.5x more but can only be held in your main hand. This suggestion makes it so staves aren't automatically better than wands and also gives a bit of variety to gameplay.

**Comment by gigaherz** [comment]
Nice suggestions. The first 3 I will certainly keep in mind. The last one, however, makes no sense.

First, because Minecraft allows anything to go in the offhand slot, so the best I could do is not allow it to work. But that wouldn't make sense either, since I already don't have any code to make them work from the offhand (and I don't have any plans currently for it). Also, I explicitly want staves to be better, that's the whole point I added them. In fact, I feel like they are too easy to craft right now, and I want to eventually make it so that players feel like they accomplished something when they gain the ability to craft the staves. I have just not decided how.

**Comment by darklordwonton** [comment]
Yeah tbh the last suggestion was just an idea I had last minute, I thought it might be interesting but now I realize it probably doesn't make much sense for this mod.
